CHANGELOG.md

```diff
@@ -36,6 +36,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Add update and delete support in pull-based ingestion ([#17822](https://github.com/opensearch-project/OpenSearch/pull/17822))
 - Allow maxPollSize and pollTimeout in IngestionSource to be configurable ([#17863](https://github.com/opensearch-project/OpenSearch/pull/17863))
 - [Star Tree] [Search] Add query changes to support unsigned-long in star tree ([#17275](https://github.com/opensearch-project/OpenSearch/pull/17275))
+- Introduce MergedSegmentWarmerFactory for Local/Remote merged segment pre copy ([#17881](https://github.com/opensearch-project/OpenSearch/pull/17881))
```

Comment:
The change log gets added in the release notes. We can review a generic statement that a user can make sense of it.